tests/api.rs

```diff
@@ -97,6 +97,16 @@ async fn test_tenants_and_users() {
     assert_eq!(tenants[0].deleted_at, None);
     assert_eq!(tenants[1].deleted_at, None);
 
+    // Verify a single tenant can be fetched by ID
+    let tenant = client.get_tenant(tenants[0].id).await.unwrap();
+    assert_eq!(tenant.id, tenants[0].id);
+
+    // Verify an unknown tenant raises a suitable error
+    let tenant_result = client
+        .get_tenant(uuid::uuid!("00000000-0000-0000-0000-000000000000"))
+        .await;
+    assert!(tenant_result.is_err());
```

Comment:
Can you extend this test to match on the expected synthetic error?

```rust
match tenant_result {
    Err(Error::Api(ApiError { status_code, .. })) if status_code == Status::NOT_FOUND => (),
    _ => panic!("unexpected response: {tenant_result:?}")
}
```

Reply:
Good call, done.